src/components/Categories/MainCategoryTile.vue

```diff
@@ -1,85 +1,85 @@
-			:images="retinaImageExists ?
-				[['small', image],
-					['small retina', retinaImage]]
-				:
-				[['small', image]]"
-			loading="lazy"
-			:alt="imageAlt"
-		/>
-		<large-tile-info
-			:category-name="categoryName"
-			:category-description="categoryDescription"
-			:number-loans="numberLoans"
-		/>
-	</div>
-	<div v-else-if="tileSize === 'medium'" class="tw-mb-6">
-		<kv-responsive-image class="category-image"
-			:images="retinaImageExists ?
-				[['small', image],
-					['small retina', retinaImage]]
-				:
-				[['small', image]]"
-			loading="lazy"
-			:alt="imageAlt"
-		/>
-		<div class="tw-hidden md:tw-block">
-			<h3 class="tw-mt-2 tw-mb-2">
-				Lend to {{ categoryName }}
-			</h3>
+	<div v-if="tileSize === 'large' || tileSize === 'medium'" class="tw-mb-6">
+		<router-link
+			class="tw-no-underline
+			active:tw-no-underline visited:tw-no-underline
+			hover:tw-no-underline focus:tw-no-underline
+			tw-text-black
+			active:tw-text-black visited:tw-text-black
+			hover:tw-text-black focus:tw-text-black"
+			:to="`/lend-by-category/${cleanURL}`"
+			v-kv-track-event="['Lending', 'click-Category', categoryName]"
+		>
+			<kv-responsive-image class="category-image"
+				:images="chooseImage(tileSize)"
+				loading="lazy"
+				:alt="altText"
+			/>
 			<div>
-				<span class="tw-line-clamp-3">
-					{{ categoryDescription }}
-				</span>
+				<div class="tw-text-center tw-flex tw-justify-between">
+					<h3 class="tw-mt-2 tw-mb-2">
+						Lend to {{ categoryName }}
+					</h3>
+					<h4
+						class="tw-mt-2 tw-mb-2"
+						:class="tileSize === 'medium' ? 'md:tw-hidden' : ''"
+					>
+						{{ numberLoans }} loans
+					</h4>
+				</div>
+				<div>
+					<p class="tw-line-clamp-3">
+						{{ categoryDescription }}
+					</p>
+				</div>
 			</div>
-			<h4 class="tw-mt-1 tw-mb-2 ">
+			<h4 v-if="tileSize === 'medium'"
+				class="tw-mt-1 tw-mb-2 tw-hidden md:tw-block"
+			>
 				{{ numberLoans }} loans
 			</h4>
-		</div>
-		<div class="md:tw-hidden">
-			<large-tile-info
-				:category-name="categoryName"
-				:category-description="categoryDescription"
-				:number-loans="numberLoans"
-			/>
-		</div>
+		</router-link>
 	</div>
 	<div v-else-if="tileSize === 'small'" class=" tw-mb-6">
-		<div class="tw-flex">
-			<div class="tw-mr-2 tw-flex-none">
-				<kv-responsive-image class="category-image-small"
-					:images="retinaImageExists ?
-						[['small', image],
-							['small retina', retinaImage]]
-						:
-						[['small', image]]"
-					loading="lazy"
-					:alt="imageAlt"
-				/>
-			</div>
-			<div class="tw-grow">
-				<h3 class=" tw-mb-2">
-					Lend to {{ categoryName }}
-				</h3>
-				<div>
-					<span class="tw-line-clamp-2">
-						{{ categoryDescription }}
-					</span>
+		<router-link
+			class="tw-no-underline
+			active:tw-no-underline visited:tw-no-underline
+			hover:tw-no-underline focus:tw-no-underline
+			tw-text-black
+			active:tw-text-black visited:tw-text-black
```

Comment:
This isn't working and the text is still kiva-green on highlight. We can tackle this in ACK-322